SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/EventTracerComponent.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Components/ActorComponent.h"
+#include "CoreMinimal.h"
+#include "EventTracerComponent.generated.h"
+
+UCLASS()
```

Comment:
is this a purely test-related component? maybe I would add `HideDropdown` in that case